frontend/src/scenes/pipeline/hogfunctions/HogFunctionInputs.tsx

```diff
@@ -240,7 +241,14 @@ function DictionaryField({ onChange, value }: { onChange?: (value: any) => void;
 export function HogFunctionInputRenderer({ value, onChange, schema, disabled }: HogFunctionInputProps): JSX.Element {
     switch (schema.type) {
         case 'string':
-            return <LemonInput value={value} onChange={onChange} className="ph-no-capture" disabled={disabled} />
+            return (
+                <CodeEditorInline
```

Comment:
The only thing that I think doesn't work here is that the autocomplete is based on the Clickhouse style event structure which is different to the structure used by HogFunctions.

Needs to be structured like the json editor https://github.com/PostHog/posthog/blob/master/frontend/src/scenes/pipeline/hogfunctions/HogFunctionInputs.tsx#L47

Reply:
Yeah, this felt like something I could push to the next PR as this got big/long enough. Will definitely fix.